src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -78,16 +79,15 @@ public void ClusterOnDatabaseChanged(object sender, string dbName)
 
                     _serverStore.NotificationCenter.Add(DatabaseChanged.Create(dbName, DatabaseChangeType.Delete));
                     
-                    //TODO: This is BAD BAD BAD thing to do since this is called on startup
-                    using (var cmd = context.ReadObject(new DynamicJsonValue
+                    ThreadPool.QueueUserWorkItem(x =>
```

Comment:
This can be dangerous pattern to use, if you are sending it with blittable members (because of ownership issues).
It's fine here.

Why use the tread pool and not just a queued async op?